---
title: "'GoTo <labelname>' non è valida perché '<labelname>' si trova all'interno di un'istruzione 'Try', 'Catch' o 'Finally' che non contiene questa istruzione"
ms.date: 07/20/2015
f1_keywords:
- bc30754
- vbc30754
helpviewer_keywords:
- BC30754
ms.assetid: 2eefc7fb-fdf0-41e9-bf60-c3bc93580e14
ms.openlocfilehash: 54fbd1852d60605290bef7816b05f7b9ffc8c673
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 06/04/2020
ms.locfileid: "84412546"
---
# <a name="goto-labelname-is-not-valid-because-labelname-is-inside-a-try-catch-or-finally-statement-that-does-not-contain-this-statement"></a>'GoTo \<labelname>' non è valida perché '\<labelname>' si trova all'interno di un'istruzione 'Try', 'Catch' o 'Finally' che non contiene questa istruzione
Non è consentito creare rami in un blocco `Try...Catch...Finally` .  
  
 **ID errore:** BC30754  
  
## <a name="to-correct-this-error"></a>Per correggere l'errore  
  
- Ristrutturare il codice inserendo l'etichetta prima del blocco `Try...Catch...Finally` .  
  
## <a name="see-also"></a>Vedere anche

- [Istruzione Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
